src/mds/MDLog.cc

```diff
@@ -581,6 +581,27 @@ class C_OFT_Committed : public MDSInternalContext {
   }
 };
 
+void MDLog::try_to_commit_open_file_table()
+{
+  ceph_assert(ceph_mutex_is_locked_by_me(submit_mutex));
+
+  if (mds_is_shutting_down) // shutting down the MDS
+    return;
+
+  if (mds->mdcache->open_file_table.is_any_committing())
+    return;
+
+  uint64_t last_seq = get_last_segment_seq();
```

Comment:
@lxbsz This is causing the MDS to crash as per the tracker you created - https://tracker.ceph.com/issues/55903

(branch wip-vshankar-testing1-20220603-134300 has these changes).

Calling this from MDLog::trim_all() with empty segments would hit an assert when calling get_last_segment_seq().

Reply:
Okay, will check it.